exporter/internal/queue/persistent_queue.go

```diff
@@ -193,31 +193,14 @@ func (pq *persistentQueue[T]) restoreQueueSizeFromStorage(ctx context.Context) (
 // The call blocks until there is an item available or the queue is stopped.
 // The function returns true when an item is consumed or false if the queue is stopped.
 func (pq *persistentQueue[T]) Consume(consumeFunc func(context.Context, T) error) bool {
```

Comment:
Is this still needed?

Reply:
Thanks! I thought I cleaned that up.. Let me remove this